utils/java/mvn_test.go

```diff
@@ -344,3 +344,43 @@ func TestRemoveMavenConfig(t *testing.T) {
 	assert.NoError(t, err)
 	assert.FileExists(t, mavenConfigPath)
 }
+
+func TestMavenDepTreeManager_suspectCurationBlockedError(t *testing.T) {
+	tests := []struct {
+		name          string
+		wantMsgToUser string
+		input         string
+	}{
+		{
+			name:          "failed on 403",
+			wantMsgToUser: "Please verify pass-through enabled on the curated repos",
+			input: "ERROR] Failed to execute goal on project my-app: Could not resolve dependencies for project com.mycompany.app:my-app:jar:1.0-SNAPSHOT: Failed to " +
+				"collect dependencies at junit:junit:jar:3.8.1: Failed to read artifact descriptor for junit:junit:jar:3.8.1: " +
+				"The following artifacts could not be resolved: junit:junit:pom:3.8.1 (absent): Could not transfer artifact junit:junit:pom:3.8.1 " +
+				"from/to artifactory (http://test:8046/artifactory/api/curation/audit/maven-remote): status code: 403, reason phrase: Forbidden (403)",
+		},
+		{
+			name:          "failed on 500",
+			wantMsgToUser: "Please verify pass-through enabled on the curated repos",
+			input: "[ERROR] Failed to execute goal on project my-app: Could not resolve dependencies for project com.mycompany.app:my-app:jar:1.0-SNAPSHOT: " +
+				"Failed to collect dependencies at junit:junit:jar:3.8.1: Failed to read artifact descriptor for junit:junit:jar:3.8.1: The following artifacts" +
+				" could not be resolved: junit:junit:pom:3.8.1 (absent): Could not transfer artifact junit:junit:pom:3.8.1 from/to artifactory" +
+				" (http://test:8046/artifactory/api/curation/audit/virtual-to-smart): status code: 500, reason phrase: Internal Server Error (500)",
```

Comment:
If you are already splitting the `input` and concatatane multiple strings, you have similar entries and you can merge them as const and concatanate thier values for readability.

Reply:
its a test, just copied the mvn cli output